spec/models/term_spec.rb

```diff
@@ -112,6 +112,15 @@
       expect(term.uuid).not_to be blank?
     end
 
+    context 'when alt_label is set' do
+      let(:term) { FactoryBot.build(:temp_term, alt_label: ['Big Foot']) }
+
+      it 'returns validation error' do
+        expect(term.save).to be false
+        expect(term.errors.full_messages).to include 'Alt label must be blank'
```

Comment:
Okay if we change this message to 'Alt label is not allowed for temporary terms' ?

Reply:
Sure.